app/controllers/notifications_controller.rb

```diff
@@ -10,6 +10,8 @@ def index
 
     post_ids = @notifications.map(&:post_id).compact_blank
     @posts = posts_from_relation(Post.where(id: post_ids), with_pagination: false).index_by(&:id)
+
+    use_javascript('global')
```

Comment:
Huh, the global javascript doesn't load everywhere? Interesting. It's been a while since I touched how we load the javascript but this surprises me